go.mod

```diff
@@ -112,7 +112,6 @@ require (
 )
 
 replace (
-	github.com/RangelReale/osin => github.com/openshift/osin v1.0.1-0.20180202150137-2dc1b4316769
```

Comment:
If you want, in your latest commit (`Updated osin to fix the uri validate issue `), you could fix the import of `osin` as well in an explicit form. Here's how I would go about it:

```shell
find pkg cmd -name '*.go' -exec sed -i 's|"github.com/RangelReale/osin"|"github.com/openshift/osin"|' {} +
go get github.com/openshift/osin@0f4d38c6e53f
go mod tidy && go mod vendor
```